examples/stable_diffusion_v2/scripts/run_controlnet_canny2image.sh

```diff
@@ -2,20 +2,21 @@ export GLOG_v=3
 export ASCEND_GLOBAL_LOG_LEVEL=3
 export ASCEND_SLOG_PRINT_TO_STDOUT=0
 
-export DEVICE_ID=1
+export DEVICE_ID=4
 export INPUT_FILE_NAME="dog2"
 #dog bird cyber dog2 house toy
 export TASK_NAME="canny2image"
-
 export MODEL_DIR="./models/"
 
-output_path=output
-mkdir -p ${output_path}
 
+rm -rf "output/"${TASK_NAME:?}
+mkdir -p "output/"${TASK_NAME:?}
```

Comment:
what is the purpose of changes here? If necessary, please modify corresponding part in segmentation control script,too.

Reply:
Create a new folder to make sure the log file and the generated result can be stored in the same folder.